e2e-tests/playwright.config.ts

```diff
@@ -0,0 +1,86 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv';
+// import path from 'path';
+// dotenv.config({ path: path.resolve(__dirname, '.env') });
```

Comment:
```suggestion
```

Reply:
It is default generated comment. We are not going to introduce env variables in current PR